docs/samples/pytorch/pytorch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "pytorch-cifar10"
+spec:
+  default:
+    pytorch:
+      modelUri: "gs://kfserving-samples/models/pytorch/cifar10/"
+      modelClassName: "Net"
```

Comment:
This feels a tad awkward. It should be implicitly derived, imo

Reply:
There can be multiple model classes in a model file in case of ensemble models.

Reply:
Yeah, not a bit but quite brittle. All the chances of that approach breaking